app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3 or command_parts[0] != "cp":
```

Comment:
Let's check only length here, and check  if first elements equals cp after we unpack varialbes to make it more self-explanatory